### PR TITLE
Changed inappropriate variable names

### PR DESCRIPTION
**Pull request by akki**
I have made the changes in two different commits. I can squash them if that would be better.

### CODE REVIEW

File: django/db/backends/sqlite3/schema.py [hunk 1]
```diff
@@ -95,7 +95,9 @@ def is_self_referential(f):
         # If any of the new or altered fields is introducing a new PK,
         # remove the old one
         restore_pk_field = None
-        if any(f.primary_key for f in create_fields) or any(n.primary_key for o, n in alter_fields):
+        if any([
+                getattr(create_field, 'primary_key', False),
```
Comment (timgraham) [inline django/db/backends/sqlite3/schema.py]: No need to use `any`... instead: `getattr(create_field, 'primary_key', False) or (a and b) ...`

Comment (akki) [inline django/db/backends/sqlite3/schema.py]: The line was exceeding 120 chars. I have updated this with one way which worked for me without raising flake errors but I am not sure if this matches our coding style.

**Comment by timgraham** [comment]
@andrewgodwin, do you see a reason to keep the looping that the second commit removes?

**Comment by andrewgodwin** [comment]
Hm, it is more efficient to do lots of those rewrites at once, but I'm not sure if the backend as it stands ever does that. It's probably fine to remove in the pursuit of simpler code.

**Pull request merged**